### ISSUE TITLE
[JENKINS-68240] Add BuildNumber as response when a Job is submitted using Jenkins API 

### ISSUE THREAD

**Issue by timja-bot** [timja-org/jenkins-gh-issues-poc-11-04#2321]
<p>We have a use-case where multiple users can submit requests and our backend needs to process all of them in parallel and continuously check for status of the running job. We did some research on how to associate a job which is submitted with the buildnumber, currently we are using nextBuildnumber concept inorder to get build number and check the status, but this might not work when multiple users are sending requests at the same time. Please let us know if there is any way to achieve this and if I need to make a feature request to get the build number when a job is submitted</p>

---
<details><summary><i>Originally reported by <a title="Shravani Ravinuthala" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=shravani777">shravani777</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-68240" target="_blank">Add BuildNumber as response when a Job is submitted using Jenkins API </a></i></summary>
<i><ul>
<li><b>status</b>: Resolved
<li><b>priority</b>: Major
<li><b>resolution</b>: Won't Do
<li><b>resolved</b>: 2022-04-23T11:03:29+00:00
<li><b>imported</b>: 2025-11-04
</ul></i>
</details>
